lib/private/App/AppStore/Fetcher/Fetcher.php

```diff
@@ -180,6 +190,10 @@ public function get($allowUnstable = false) {
 				return $responseJson['data'];
 			}
 
+			if (empty($responseJson['data'])) {
```

Comment:
Shouldn't it be 
```suggestion
			if (empty($responseJson)) {
```

?

Reply:
Also should be before:
```
if ($allowUnstable) {
```

Reply:
Worked as well, but the empty check on the whole responseJson makes more sense of course.

Fixed and pushed adjusted tests as well.